test/cypress/integration/signup-and-login.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('SignUp, Profile and Login', () => {
+  // NOTE: Create a uniqueId to avoid duplicated users on DB during crypress watch mode (during dev)
+  const userId = new Date().getMilliseconds()
+  const userName = `user1-${userId}`
+
+  it('successfully loads homepage', () => {
+    cy.visit('/')
+  })
+
+  // NOTE: What's the purpose of this test?
+  // it.skip('Should start the backend server if necessary', function () {
+  //   return require('../backend/index.js')
+  // })
```

Comment:
@taoeffect This test was copied from the Nightmare tests but didn't understand why it was needed.

Reply:
This is because mocha was being used to run both the frontend and backend tests. If both frontend and backend tests were being run, the backend tests would have already started the server and this would do nothing. If only the frontend tests were being run (because we were using `describe.only` on the frontend tests), then this would start up the backend API server. So by having this here we would ensure the backend API server was running.

Reply:
so now we don't need it. removed.